config/jobs/navarchos/navarchos-presubmits.yaml

```diff
@@ -0,0 +1,154 @@
+job_template: &job_template
+  max_concurrency: 10
+  path_alias: github.com/pusher/navarchos
+  clone_uri: git@github.com:pusher/navarchos
+  agent: kubernetes
+  always_run: true
+  skip_report: false
+  decorate: true
+
+container_template: &container_template
+  image: quay.io/pusher/kubebuilder-builder:v20190821-328974b
+  name: runner
+  command: ["/usr/local/bin/runner"]
+
+container_template_small: &container_template_small
+  <<: *container_template
+  resources:
+    requests:
+      cpu: 1
+      memory: 1Gi
+    limits:
+      cpu: 2
+      memory: 2Gi
+
+container_template_large: &container_template_large
+  <<: *container_template
+  resources:
+    requests:
+      cpu: 2
+      memory: 4Gi
+    limits:
+      cpu: 4
+      memory: 8Gi
```

Comment:
The nodes we're using only have 7.2Gi of RAM available, so this limit will never be reached.

Reply:
Ok, I'll change it to `6Gi`